packages/babel-types/src/clone/cloneNode.js

```diff
@@ -27,8 +27,14 @@ function cloneIfNodeOrArray(obj, deep) {
 /**
  * Create a clone of a `node` including only properties belonging to the node.
  * If the second parameter is `false`, cloneNode performs a shallow clone.
+ * If the third parameter is `true`, cloneNode performs a shallow clone
+ * of a `node` excluding `_private` location properties.
```

Comment:
Nit: "If the third parameter is `true`, the cloned nodes exclude location properties."